src/background/tabHandler.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {Component} from "./component.js";
+import {Logger} from "./logger.js";
+
+const log = Logger.logger("TabHandler");
+
+export class TabHandler extends Component {  
+  constructor(receiver) {
```

Comment:
i think you can just remove the constructor in case you don't change anything, right?